apiserver/common/crossmodel/crossmodel.go

```diff
@@ -190,11 +190,11 @@ func handleChangedUnits(change params.RemoteRelationChangeEvent, applicationTag
 func GetOfferingRelationTokens(backend Backend, tag names.RelationTag) (string, string, error) {
 	offerName, err := backend.OfferNameForRelation(tag.Id())
 	if err != nil {
-		return "", "", errors.Trace(err)
+		return "", "", errors.Annotatef(err, "error getting offer for relation %q", tag.Id())
```

Comment:
do we need the "error "?

Reply:
can take it or leave it - there's several other places that include it; the other common one to use is "cannot ..."

Reply:
removed the "error"